fastlane/Fastfile

```diff
@@ -58,7 +58,16 @@ platform :android do
       description: changelog,
       commitish: "main",
       upload_assets: []
-  )
+    )
+    add_git_tag(
+      tag: "latest",
+      force: true,
+      grouping: "releases"
```

Comment:
@vegaro @beylmk 
this could lead to some unexpected behavior: what if you patch an old version of the SDK? 
i.e.: we're at 4.2.3 but we issue a hotfix for 3.4.2
I think we need to: 
- check out the existing "latest" tag's version number
- check that it is indeed higher
- only update the tag if it's higher

Reply:
Good point. It's kinda complex though... I think we could do something like this too:

- Get "latest" tag's message, which should indicate the version
- Check that it's higher
- Update "latest" tag including the new version in the message

I hope this http://docs.fastlane.tools/actions/last_git_tag/ returns the message of the tag

Reply:
👍